docs/src/further_topics/ugrid/data_model.rst

```diff
@@ -0,0 +1,328 @@
+structured grid, including:
+
+* `The UK Met Office's LFRic cubed-sphere <https://hps.vi4io.org/_media/events/2018/sig-io-uk-adams.pdf>`_
+* `Oceanic model outputs <https://doi.org/10.3390/jmse2010194>`_
+
+.. todo:
+        a third example!
+
+Mesh 'payload' is much larger than with structured grids
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Coordinates are recorded per-node, and connectivities are recorded per-element.
+This is opposed to a structured grid, where a single coordinate value is shared
+by every data point/area along that line.
+
+For example: representing a cubed-sphere using a mesh leads to coordinates and
+connectivities being **~8 times larger than the data itself**, as opposed to a
+small fraction of the data size when dividing a sphere using a structured grid.
+
+This further increases the emphasis on lazy loading and processing of data
+using packages such as Dask.
+
+.. note::
+
+        The large, 1D data arrays associated with meshes are a very different
+        shape to what Iris users and developers are used to. It is suspected
+        that optimal performance will need new chunking strategies, but at time
+        of writing (``Jan 2022``) experience is still limited.
+
+.. todo:
+        Revisit when we have more information.
+
+Spatial operations on mesh data are more complex
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Detail: :doc:`operations`
+
+Indexing a mesh data array cannot be used for:
+
+#. Region selection
+#. Neighbour identification
+
+This is because - unlike with a structured data array - relative position in
+a mesh's 1-dimensional data arrays has no relation to relative position in
+space. We must instead perform specialised operations using the information in
+the connectivities present, or by translating the mesh into a format designed
+for mesh analysis such as VTK.
+
+Such calculations can still be optimised to avoid them slowing workflows, but
+the important take-away here is that **adaptation is needed when working mesh
+data**.
+
+
+How Iris Represents This
+========================
+
+..
+    Include API links to the various classes
+
+    Include Cube/Mesh printout(s)
+
+.. seealso::
+
+        Remember this is a prose summary. Precise documentation is at:
+        :mod:`iris.experimental.ugrid`.
+
+.. note::
+
+        At time of writing (``Jan 2022``), neither 3D meshes nor 3D elements
+        (volumes) are supported.
+
+The Basics
+----------
+The Iris :class:`~iris.cube.Cube` has several new members:
+
+* | :attr:`~iris.cube.Cube.mesh`
+  | The :class:`iris.experimental.ugrid.Mesh` that describes the
+    :class:`~iris.cube.Cube`\'s horizontal geography.
+* | :attr:`~iris.cube.Cube.location`
+  | ``node``/``edge``/``face`` - the mesh element type with which this
+    :class:`~iris.cube.Cube`\'s :attr:`~iris.cube.Cube.data` is associated.
+* | :meth:`~iris.cube.Cube.mesh_dim`
+  | The :class:`~iris.cube.Cube`\'s **unstructured dimension** - the one that
```

Comment:
This is the first use of the phrase "unstructured dimension" and is near the bottom of the page. Can we define this earlier (it might make some of the explanation easier to define this early then use it, and will make it less unexpected here)?

Reply:
3c0e5f85bff8e008226dc9d8a88da47b9b6f4db1

Reply:
> represented by either the coordinate or connectivity arrays detailed above

This bit of the `important` block doesn't make sense to me. My attempt to make this make sense below:

Elements are enumerated along a single unstructured dimension (present in the coordinate or connectivity arrays detailed above). An element’s position along this dimension has nothing to do with its spatial position.

Reply:
a31546293bde29313aa7d24dd647ac27476c6604